x/axelarcork/keeper/abci.go

```diff
@@ -28,10 +30,26 @@ func (k Keeper) EndBlocker(ctx sdk.Context) {
 				"winning votes", winningScheduledVotes,
 				"chain id", config.Id)
 			for _, c := range winningScheduledVotes {
-				k.SetWinningAxelarCork(ctx, config.Id, c)
+				k.SetWinningAxelarCork(ctx, config.Id, uint64(ctx.BlockHeight()), c)
 			}
 		}
 
+		k.Logger(ctx).Info("removing timed out approved corks",
+			"height", fmt.Sprintf("%d", ctx.BlockHeight()),
+			"chain id", config.Id)
+
+		timeoutHeight := k.GetParamSet(ctx).CorkTimeoutBlocks + uint64(ctx.BlockHeight())
```

Comment:
Shouldn't this be current block height subtracting the timeout param? Adding it to the current block height will always produce one in the future.

Reply:
My bad, fixed